docs/application/toc_all.md

```diff
@@ -84,8 +84,7 @@
 ##### [Create Custom Layout](/application/dotnet/guides/nui/custom-layout.md)
 #### Components
 ##### [Overview](/application/dotnet/guides/nui/ui-components.md)
-##### [Styling UI Components](/application/dotnet/guides/nui/styling-controls-with-JSON.md)
```

Comment:
If you are removing this file from toc as for me it seems that the content is deprecated or no longer needed, shouldn't the file **styling-controls-with-JSON.md** be removed?
https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/guides/nui/styling-controls-with-JSON.md

Reply:
I removed the file: styling-controls-with-JSON.md.
(it is deprecated: https://code.sec.samsung.net/jira/browse/TCSACR-410)